ibis_mssql/tests/test_ops.py

```diff
@@ -0,0 +1,26 @@
+from ibis.tests.expr.mocks import MockConnection
+import unittest
+import pytest
+
+pytest.importorskip('sqlalchemy')
```

Comment:
as sqlalchemy is a required library for ibis-mssql, probably we don't want this "skip" here 
ref https://github.com/Quansight/ibis-mssql/blob/master/setup.py#L31

Reply:
Yes.  Got it! ... Done.